### DISCUSSION TITLE
FM burner simulations

### DISCUSSION THREAD

**Discussion by NingRen** [MaCFP/macfp-db#450] [General]
Hi Everyone,

I am one of the section chairs for the FM burner case and thanks everyone for contributing the modeling results. I'd like to reach out to clarify some questions about model details and create this thread to facilitate discussions. Please cc any coauthors if I missed in this thread. The authors I found on github are
@rmcdermo 
@mcgratta 
@drjfloyd 
@gmaragko82 
@ShMotaghian 
@tbeji
@fbraennstroem 
@hong27 

We have a total of 5 submissions. Can you please share some images of case/mesh setup and the rendered flame structure (at normal air condition, C2H4)?

I have some clarification questions,
1. UGent and UNSW (@gmaragko82, @tbeji, @hong27) used dynamic model. Is there any averaging treatment for the dynamic coefficient? Do you see backscattering or any stability issue related with the turbulence model?
2. For the combustion model, there are not much information about extinction/re-ignition. Could you provide more info or share some references about the model you used?
3. Some references or more details about the soot model are also appreciated.
4. For the radiation and radiant fraction, NIST, UGent2 and UWuppertal used predicted radiant fraction based on RadCal tabulated absorption coefficient; UGent and UNSW used WSGG model. When you do the absorption/emission calculations, do you use the cell temperature with some TRI treatment? UGent (@gmaragko82 ) mentioned the radiant fraction is predicted with some corrections from measurement. Could you provide some details about this method?

Best Regards!

Ning

**Comment by gmaragko82** [comment]
Hi Ning,

You can find below some information for the UGent submission.
 
Below is a photo of the 5 mm mesh that was used in the simulations showing the instantaneous and average gas-phase temperatures. No local grid refinement was used for the 2 cm cases.

![UGent_FM_Burner_mesh](https://github.com/MaCFP/macfp-db/assets/59868136/8ccd07e2-a8f7-4cf5-82db-53b10e10214b)
![UGent_FM_Burner_mesh2](https://github.com/MaCFP/macfp-db/assets/59868136/20380f3f-7495-42de-aac1-e378051b32cc)

1. The dynamic turbulence model coefficients were calculated as local averages of their face values. Sgs viscosity was allowed to take negative values such that mu_eff >=0 and negative mu_sgs values were observed above the burner. On the centerline and up to a height of 0.2 m from the burner, the average mu_sgs value was mostly either negative or close to 1e-6 (for the 5 mm grid size cases). A similar behavior was observed for the 2 cm grid size cases but up to a height of 0.1 m. No stability problems were encountered.

2. Extinction/re-ignition was essentially treated together as an ignition problem, with a very simple model, where combustion was allowed to take place if the local cell temperature was equal or above an ignition temperature. The ignition temperatures were taken from counter-flowing experiments for different fuels and varied locally as a function of the sgs strain rate. More details about the extinction approach and the combustion model (EDM) are given in https://doi.org/10.1016/j.firesaf.2023.103952.

3. Soot modelling was not considered.

4. The absorption/emission calculations in WSGG used the cell temperatures while no TRI was considered. To account for any under-predictions in the radiative fraction a simple approach was considered. During the simulation, the predicted radiative fraction is calculated (chi_pred.) and if it is under-predicted based on the experimental radiative fraction (chi_exp) then a correction in the form of (chi_exp – chi_pred.) x q_dot is applied in every cell. In scenarios without flame extinction, chi_exp was a constant value while for the extinction scenarios, chi_exp., decreased based on available experimental data (i.e., taken from the FM burner and UMD line burner scenarios). Some more details regarding the approach and the WSGG model are given in https://doi.org/10.1016/j.firesaf.2023.103952.

If it’s of interest, a video from an extinction scenario with C2H4 is available in https://youtu.be/ya0yY6Pi88c.

Regards,
Giorgos

**Reply by NingRen** [reply]
Hi Giorgos,

@gmaragko82 Thank you for these clarifications. Looks great! I'll read your paper to get more info. 

What do you use for flame stabilization at the base? For the published experimental data, we used a ring of pre-mixed C2H4 flames (about 1 kW) to stabilize the flame. Recently, we used electric heating coil to replace the pre-mixed flame and found the combustion efficiency shifts a little bit. Gang will briefly mention this in the workshop presentation and hopefully we can release these new data soon.

Regards,

Ning

**Reply by gmaragko82** [reply]
Sure Ning, let me know if you need any further clarification. I didn't do anything regarding flame stabilization at the base (no pilot flames were used). I should mention, however, that no extinction was allowed in a small region just above the burner (up to a height of 2 cm) in order not to fully suppress the primary source of combustion.

Regards,
Giorgos

**Comment by drjfloyd** [comment]
Ning,

@rmcdermo is working on some images for you but here are answers to your questions:

1. N/A
2. In LES mode FDS allows combustion to take place if a grid cell meet two conditions: 1) The cell temperature exceeds the AUTO_IGNITION_TEMPERATURE and 2) if the mixed cell contents when burnt raise the temperature of the products above a CRITICAL_FLAME_TEMPERATURE. The model is presented in 5.3.3 and 5.3.4 of the FDS Technical Reference Guide.
3. Two-step chemistry was used where the first step oxidizes the fuel to H2O, CO, and C and the second second step oxidizes the CO and C to CO2. The ratio of CO to C in the first step is specified by an input. For the second step, fixed post-flame yields for CO and C define how much of the CO and C from the first step can go to CO2 4. Only the cell temperature is used.
4. Cell temperature was used without a TRI model.

**Reply by rmcdermo** [reply]
Here is an image of the mesh set up for the "5 mm" case (red refined region).  For the cases where we are testing the LOI for extinction, by ramping the inlet N2 flow rate, we use a ring of "pilot fuel particles", as shown in the image.  These are trying to mimic the original 36 premixed ethylene pilot flames at 1 kW.  The cases for soot and radiation do not use the pilot and are at a fixed O2 level.  Also attached below is a video of the 15.2 vol% O2 ethylene flame.  You will notice that the flame structure is not quite right.  Our "puffs" are rolling up faster than in the flame images in the FM burner documentation.  I suspect several issues, one is again the new geometry routines and the computation of the vorticity at edges for cutcells.  Other possible issues: extinction model, flame chemistry right at the base, burner surface temperature.  In some of the 5 mm plots of radial profiles of mean soot concentration, you can see the effect of the symmetric roll up of the puff right at the level of the profile, and this happens consistently enough to affect the mean for the small time window we are using. (OK, soot is too high in the center, but the point of the plot is to illustrate the  effect of the symmetric puff.)

mesh setup for "5mm" resolution:
<img width="400" alt="Screenshot 2023-09-19 at 11 38 58 AM" src="https://github.com/MaCFP/macfp-db/assets/4418497/7f1065eb-2cdb-4a57-9c05-19e9d27b107b">

video of ethylene 15.2 vol% O2:
https://github.com/MaCFP/macfp-db/assets/4418497/f13cb83c-dfd1-4a77-bf34-70eb0948a857

mean soot profile at 1D:
<img width="400" alt="Screenshot 2023-09-19 at 11 57 08 AM" src="https://github.com/MaCFP/macfp-db/assets/4418497/c3f05f85-ebcc-4f31-b86e-4e5d00b28e58">

**Reply by NingRen** [reply]
Hi Jason and Randy,

@drjfloyd, @rmcdermo 
Thanks for providing these information. I have one quick question for the soot prediction. The predicted value is several times higher than the measurement. I assume that you can tune the soot yield fraction to better match the measurement. I guess the reason you didn't do that is to better predict the radiant fraction. Is that right? I am wondering when a TRI model is adopted, it may be possible to improve the soot prediction, while maintain a good radiation prediction.

Thanks again!

Ning

**Reply by rmcdermo** [reply]
Hi @NingRen ,
This is a good question.  You are correct, if we dial back the in-flame soot yield, then our radiative emission is lower.  But I do not think this is because of TRI.  If you look at the RMS temp profiles we are doing ok, if anything we are high with  5 mm resolution at the lower z elevations.  I actually did not think our temperature PDFs were that bad.  Where we seem to be off is in the mean temperature higher up in the flame.  There I suppose that any number of things could be at fault: too much radiation down low, model for kappa (here, yes, TRI may be involved, but temperature self-correlation is usually the biggest issue[?]), extinction model?  Without knowing whether we have the heat release correct as a function of height, it is hard for me to know.  I scanned Gang's paper on LII and I did not see any mention of calorimetry (maybe I missed it), but did you all have calorimetry on the flames?

**Reply by xiongg06** [reply]
Hi @NingRen @rmcdermo , 
About NIST's soot prediction, although the profile at 1D is too high in the center, the prediction at 2.5D looks very good. That means if we dial back the soot yield, the SVF at 2.5D will likely be much lower. Note that the radiation along height drops faster than experiments, it seems that there is too much reaction close to the burner and therefore too much soot production at lower heights.

**Reply by rmcdermo** [reply]
Not sure that is entirely true to say the soot concentration at 2.5D will be "much lower".  At some point, the post-flame yield starts controlling the soot concentration.  Here are the 20.9% mean soot volume fraction plots side by side.

<img width="250" alt="Screenshot 2023-09-20 at 10 31 23 AM" src="https://github.com/MaCFP/macfp-db/assets/4418497/77c0c369-3d3e-433b-a722-cc93858ef908">
<img width="250" alt="Screenshot 2023-09-20 at 10 30 20 AM" src="https://github.com/MaCFP/macfp-db/assets/4418497/f5411736-b969-463e-b9e1-ea440ed426e4">
<img width="250" alt="Screenshot 2023-09-20 at 10 29 42 AM" src="https://github.com/MaCFP/macfp-db/assets/4418497/84fba810-7612-40fc-8e1a-394a8e6b22ff">

But, I agree, to explain the high soot levels in the near field of the burner we either have to fault the reaction rate (mixing time scale and turbulent Schmidt number) or the FUEL_C_TO_CO_FRACTION parameter.

**Comment by ShMotaghian** [comment]
Hi Ning,

Here is some information regarding UGent2 simulations of the FM-Burner.

Some pictures of the mesh and instantaneous flame and soot stream structures constructed using temperature and soot volume fraction fields.

-Picture of the grid for the finest mesh:
	 
![image](https://github.com/MaCFP/macfp-db/assets/68729221/cb16a979-b5ba-46f6-be8c-1e76c86b88da)

-	Flame and the soot stream structures
 
![image](https://github.com/MaCFP/macfp-db/assets/68729221/ea4255c5-e85b-450e-be78-7c6227c7c3ed)

1.	The dynamic model used (compressible dynamic k-equation) does not require post-calculation averaging to maintain numerical stability. The minimum turbulent Prandtl number allowed in the domain was set to 0.05 to avoid numerical instabilities. No significant backscattering was observed.

2.	No extinction/ignition modeling was applied. However, the consumption of fuel and oxygen and the production of CO2 in the nucleation and evolution of soot particles were considered using a set of virtual soot reactions. This caused combustion efficiencies below unity in the simulations. 

3.	The soot model and the turbulence-soot interaction (TSI) model used in the simulations were developed by the authors:
  -	 The laminar soot model is a laminar smoke point-based two-equation model.
  -	The TSI model is developed based on the Eddy Dissipation Concept with a correction scheme for low-Reynolds number (nearly laminar) reacting flows.
  -	In situ adaptive tabulation (ISAT) scheme is configured based on the ordinary differential equation of the TSI model to improve numerical robustness and reduce the computational cost of soot modeling in the simulations.

The following references are related:

[1]	T. Beji, J.P. Zhang, W. Yao, M. Delichatsios, A novel soot model for fires: Validation in a laminar non-premixed flame, Combust. Flame 158 (2011) 281–290.
[2] S.J. Brookes, J.B. Moss, Predictions of soot and thermal radiation properties in confined turbulent jet diffusion flames, Combust. Flame 116 (1999) 486–503.
[3]	D.A. Lysenko, I.S. Ertesvåg, Reynolds-Averaged, Scale-Adaptive and Large-Eddy Simulations of Premixed Bluff-Body Combustion Using the Eddy Dissipation Concept, Flow, Turbul. Combust. 100 (2018) 721–768.
[4]	S.B. Pope, Computationally efficient implementation of combustion chemistry using in situ adaptive tabulation, Combust. Theory Model. 1 (1997) 41–63.

4.	No TRI treatment was applied and resolved temperature and mass fractions of CO2 and H2O were used in the calculation of terms present in the RTE. 

Best regards,
Shahrooz

**Reply by NingRen** [reply]
Hi Shahrooz 
@ShMotaghian, thanks for sharing these references and the nice plots. Looks great! The mesh is too fine in the center to see the details. Do you have a square-shaped region in the center?
Best Regards,
Ning

**Reply by ShMotaghian** [reply]
Hi Ning,

Here are some other pictures of the finest grid. Hope these are clear. Please let me know if you need more details about the simulations.

![Picture2](https://github.com/MaCFP/macfp-db/assets/68729221/00d76481-db08-458c-b227-0e4f02ee7f4f)

Best regards,

Shahrooz

**Reply by NingRen** [reply]
Thanks Shahrooz! @ShMotaghian Looks great!
Ning

**Comment by hong27** [comment]
Hi Ning, 

Here is some information regarding the UNSW simulations. 

Figures of domain/mesh setup:
![image](https://github.com/MaCFP/macfp-db/assets/49507355/24eca083-7d7e-4a7d-b448-6bab7053eb67)
![image](https://github.com/MaCFP/macfp-db/assets/49507355/d0fe3bc6-c0ac-4f33-aaa2-2d6f27fcad1b)

Figures of the instantaneous flame structure:
![image](https://github.com/MaCFP/macfp-db/assets/49507355/dd9d7ffa-f788-4b87-a1a6-78b77983a446)

Figures of mean fields : temperature, soot volume fraction and soot density number:
![image](https://github.com/MaCFP/macfp-db/assets/49507355/7869067b-921f-4c70-afae-960e71f19414)


1. We used the dynamic kEqn model readily available in OpenFOAM-7 [1]. No averaging treatment was applied. We did not observe significant instability or backscattering.
2. The combustion model applied in our work is a radiative flamelet/progress variable model. In such a model, the unstable flamelet solutions filling the gap between the burning and the mixing solutions can be introduced, leading to its capability of accounting for extinction to certain extent. A schematic of the flamelet solutions included in the flamelet table is shown below in the S-curve plot. This model was first proposed by Ihme and Pitsch [2], and we have tested it in fire-relevant conditions; some details can be found in this [paper](https://doi.org/10.1016/j.firesaf.2023.103970). However, in this workshop, only the normal air case is submitted; we won't expect the capability of predicting extinction to make a significant impact in the normal air case.
![image](https://github.com/MaCFP/macfp-db/assets/49507355/83ce2a83-8381-43ff-a258-2288c5e6174f)

3. In terms of the soot model, we incorporate a two-equation soot model (soot mass fraction and soot number density) into the radiative flamelet/progress variable framework. The soot model considered acetylene and benzene as soot precursors. In this soot model, source terms for soot nucleation, coagulation, surface growth and oxidation are according to the expression in Lindstedt's work in the book Ref. [3], which also can be found in the supplementary materials in Nmira's recent work [4]. In LES, these source terms are closed using the presumed filtered density function (FDF) method by assuming that the FDFs of gas-phase and soot quantities are independent. Please let me know if you need any other details, e.g. regarding the detailed version of the expressions. 

4. The radiation model in our simulations is actually a grey model, with the mixture-averaged absorption coefficient of CO2 and H2O curve-fit from RADCAL data, similar to Ref. [5] from Sandia. The absorption coefficient of soot is considered as the kappa_soot = C_soot * fv * T. This has been shown in our README.md file. We considered TRI by a presumed FDF method, as detailed in our [paper](https://doi.org/10.1016/j.firesaf.2023.103970) [6]. In our previous work [6], we have shown that the T-self correlation, as well as the kappa-T correlation, plays a much more significant role in TRI than the kappa-self correlation. 



[1] Kim, W and Menon, S. (1995). A new dynamic one-equation subgrid-scale model for large eddy simulation. In 33rd Aerospace Sciences Meeting and Exhibit, Reno, NV, 1995.
[2] Ihme, M., & Pitsch, H. (2008). Modeling of radiation and nitric oxide formation in turbulent nonpremixed flames using a flamelet/progress variable formulation. Physics of Fluids, 20(5).
[3] Bockhorn, H. (Ed.). (2013). Soot formation in combustion: mechanisms and models (Vol. 59). Springer Science & Business Media.
[4] Nmira, F., Bouffard, A., Consalvi, J. L., & Wang, Y. (2023). Large-eddy simulation of lab-scale ethylene buoyant diffusion flames: effects of subgrid turbulence/soot production interaction and radiation models. Proceedings of the Combustion Institute, 39(3), 3959-3968.
[5] Domino, S. P., Hewson, J., Knaus, R., & Hansen, M. (2021). Predicting large-scale pool fire dynamics using an unsteady flamelet-and large-eddy simulation-based model suite. Physics of Fluids, 33(8).
[6] Lin, J., Zhou, H., Hawkes, E. R., Ma, M. C., & Yeoh, G. H. (2023). Numerical investigation on turbulence-radiation interaction in the UMD turbulent line fires. Fire Safety Journal, 103970.

Regards,
Jianhong

**Reply by NingRen** [reply]
Hi Jianhong,
@hong27 This is great! Thanks for the very detailed explanation. I have all I need.
Best Regards,
Ning

**Comment by fbraennstroem** [comment]
Hi Ning,
excuse me for the delay... below you find 4 snapshots for the v4 and v8 setups; these differ in terms of radiation solver increment setting v4 every time step; v8 every thousands time step. So the snapshots for the different snapshots for the temperature will not provide so much information, but I am not able to create mean fields, so I post the snapshots here.
If the quality needs to be better I can email separately.
Thank you very much!
Best Regards
Fabian

### v4 dt 1; 512 solid angles
![v4_all_visit0004](https://github.com/MaCFP/macfp-db/assets/83709063/05f2ed3f-2a4e-4086-8da5-4aa20e63e004)
![v4_close_visit0003](https://github.com/MaCFP/macfp-db/assets/83709063/feab2064-564b-457b-82a0-ac40429c9c1e)

### v8 dt 1000; 512 solid angles
![v8_all_visit0001](https://github.com/MaCFP/macfp-db/assets/83709063/30ac2358-294c-49c4-a8df-a3c9378dd85e)
![v8_close_visit0002](https://github.com/MaCFP/macfp-db/assets/83709063/0b489c73-6e5d-41ee-bbe1-b5019138f80a)

**Reply by NingRen** [reply]
@fbraennstroem 
Thanks Fabian for sharing these plot! The image quality is very good. 
Best,
Ning
